src/ipc/ipc.cpp

```diff
@@ -214,6 +214,7 @@ bool is_step_collision_free(
     const Eigen::MatrixXd& V0,
     const Eigen::MatrixXd& V1,
     const BroadPhaseMethod method,
+    const double min_distance,
```

Comment:
you should also set the inflation radius in `construct_collision_candidates` to `min_distance/2`